app/views/schools/change_schools/show.html.erb

```diff
@@ -29,6 +33,7 @@
         <p class="govuk-body-l">
           You do not have access to any schools yet.
         </p>
+            <%= render 'pages/request_organisation_access' %>
```

Comment:
line 36 looks to be a duplicate of line 38?

Reply:
Good spot!

@MylesJarvis Will leave this to you to resolve on your return

<img width="662" alt="image" src="https://github.com/DFE-Digital/schools-experience/assets/87642394/c130902e-2177-4f76-a051-a50868f654d9">

Reply:
Excellent catch, pushed fix

Reply:
![image](https://github.com/DFE-Digital/schools-experience/assets/35870975/bd4a82ce-33e2-47c0-86bd-e3e81fb4ed83)

@martyn-w this is fixed.